docs/spec.md

```diff
@@ -5469,22 +5490,25 @@ where `result_index[d] = operand_index[permutation[d]]`.
 
 #### Inputs
 
-| Label | Name          | Type                                         | Constraints |
-|-------|---------------|----------------------------------------------|-------------|
-| (I1)  | `operand`     | tensor or per-tensor quantized tensor        | (C1-C3)     |
-| (I2)  | `permutation` | 1-dimensional tensor constant of type `si64` | (C2), (C3)  |
+| Label | Name          | Type                                         | Constraints   |
+|-------|---------------|----------------------------------------------|---------------|
+| (I1)  | `operand`     | tensor or quantized tensor                   | (C1-C5)       |
+| (I2)  | `permutation` | 1-dimensional tensor constant of type `si64` | (C2-C3), (C5) |
 
 #### Outputs
 
-| Name     | Type                                  | Constraints |
-|----------|---------------------------------------|-------------|
-| `result` | tensor or per-tensor quantized tensor | (C1), (C3)  |
+| Name     | Type                       | Constraints |
+|----------|----------------------------|-------------|
+| `result` | tensor or quantized tensor | (C1), (C3)  |
 
 #### Constraints
 
 * (C1) `element_type(operand) = element_type(result)`.
 * (C2) `permutation` is a permutation of `range(rank(operand))`.
 * (C3) `shape(result) = dim(operand, permutation...)`.
+* (C4 `is_per_tensor_quantized(operand) = is_per_tensor_quantized(result)`.
+* (C5) If `is_per_axis_quantized(result)`, then `quantized_dimension(result) =
+       permutation(quantized_dimension(operand))`.
```

Comment:
Seems to be the reverse:

`quantized_dimension(operand) = permutation(quantized_dimension(result))`

because according to the code above:

```
result_index[d] = operand_index[permutation[d]]
```

`permutation` is actually a result-axis to operand-axis function (or the array index is the result-axis and the array value is the operand-axis).  so we should put `quantized_dimension(result)` in `permutation[...]`.

Reply:
That is good catch! Thanks. 
Updated the spec accordingly.